en/docs/style-test.md

```
@@ -0,0 +1,43 @@
+Applying a policy is done by the administrator, it is not done by the API publisher. The
+following options aren't currently supported: request-body limits, per-user quotas and
+geo-based rules.
+
+| Field | Description |
+|---|---|
+| Quota | The request count |
+| Burst | The burst ceiling |
+
+For the deprecated fields, see the table above.
+
+<img src="../assets/img/img1.png" width="700" height="420"
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)